src/Mobs/Monster.cpp

```diff
@@ -397,32 +347,29 @@ void cMonster::SetPitchAndYawFromDestination()
 	{
 		if (m_Target->IsPlayer())
 		{
-			FinalDestination.y = ((cPlayer *)m_Target)->GetStance();
+			FinalDestination.y = static_cast<cPlayer *>(m_Target)->GetStance() - 1;
```

Comment:
Thanks! I think the FinalDestination variable is badly named in that it is separate from m_FinalDestination - it's used purely in this function for mob rotation/pitch calculations.